hotshot-query-service/src/lib.rs

```diff
@@ -447,7 +447,11 @@ use task::BackgroundTask;
 use tide_disco::{method::ReadState, App, StatusCode};
 use vbs::version::StaticVersionType;
 
-pub type VidCommon = Option<hotshot_types::vid::advz::ADVZCommon>;
+#[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq)]
+pub enum VidCommon {
+    V0(hotshot_types::vid::advz::ADVZCommon),
```

Comment:
so we bincode serialize the `VidCommon` when inserting data. This would mean, we require a migration for vid as well in query service

Reply:
basically, just changing VIDCommon serialization in the vid migration that we already have

Reply:
Yes I wasn't sure, should I add this to the migration in this PR?

Reply:
should be a few lines change. I can do it in a separate PR